### DISCUSSION TITLE
Stencil or React to Mitosis Generator

### DISCUSSION THREAD

**Discussion by markusbkk** [BuilderIO/jsx-lite#967] [General]
Curious. Does something like this exist?

I'm getting increasingly frustrated with our in-house Stencil component library (the React target works nicely but Angular is perpetually broken) and would love to have a quick fix at replacing it without rewriting everything from scratch in Mitosis.

**Comment by samijaber** [comment]
See https://github.com/BuilderIO/mitosis/issues/149#issuecomment-1353172744
